satellite/rewards/display.go

```diff
@@ -78,9 +80,18 @@ func (offers Offers) OrganizeOffersByType() OfferSet {
 func createPartnerSet() PartnerSet {
 	partners := LoadPartnerInfos()
 	var ps PartnerSet
-	for _, partner := range partners {
+	keys := make([]string, len(partners))
+	i := 0
+	for k := range partners {
+		keys[i] = k
+		i++
+	}
+
+	sort.Strings(keys)
+
+	for _, key := range keys {
```

Comment:
why not range partners here and just set to the key

Reply:
because the sort is needed, I think.  another option would be https://gobyexample.com/sorting-by-functions